.github/workflows/crowdin-upload.yml

```
@@ -0,0 +1,26 @@
+name: Crowdin upload
+
+on:
+  push:
+    branches-ignore:
+      - 'dependabot/**'
+      - 'dependabot-**'
```

Similarly only upload from master, not branches.
```suggestion
on:
  push:
    branches:
      - master
```